src/components/grapher/Grapher.vue

```diff
@@ -114,16 +133,14 @@ export default Vue.extend({
       return this.$store.getters.getBackHashOffsetY + this.$store.getters.getFrontHashOffsetY;
     },
     fourStepGridOffsetsX(): number[] {
-      // Do not render a vertical line if there is a yard line
+      let offset: number = (this.$store.state.yardlines ? 8 : 4);
```

Comment:
Added this instead of v-else-if. Thoughts?

Reply:
Nice! This is probably a better change. Note that you can get rid of the parenthesis here.
```suggestion
      let offset: number = this.$store.state.yardlines ? 8 : 4;
```